docs/notebooks/expected_improvement.pct.py

```diff
@@ -125,20 +125,20 @@ def build_model(data):
 # We can now get the best point found by the optimizer. Note this isn't necessarily the point that was last evaluated.
 
 # %%
-query_points = dataset.query_points.numpy()
-observations = dataset.observations.numpy()
-
-arg_min_idx = tf.squeeze(tf.argmin(observations, axis=0))
+query_point, observation, arg_min_idx = result.try_get_optimal_point()
```

Comment:
this can probably be used in other notebooks as well, can we propagate it there as well?